public/styles/colors.css

```diff
@@ -93,6 +93,9 @@
     --box-shadow-small: 0 1px 3px rgba(0, 0, 0, 0.2);
     --box-shadow-medium: 0 1px 8px rgba(0, 0, 0, 0.2);
     --box-shadow-big: 0 12px 18px rgba(0, 0, 0, 0.12), 0 5px 22px rgba(0, 0, 0, 0.16), 0 7px 8px rgba(0, 0, 0, 0.2);
+
+    /*** Options Switcher ***/
+    --box-shadow-open-switcher: 0 1px 4px rgba(0, 0, 0, 0.3);
```

Comment:
I'm not sure we want a feature-specific color in our design system. If that's just a special use case I'd just use it in the app and not add it to the DS